test/actions/test_images_error.ts

```
@@ -0,0 +1,112 @@
+    },
+    presentationStyle: studio.script,
+  };
+  return context;
+};
+
+test("test images", async () => {
+  // const fileDirs = getFileObject({ file: "hello.yaml", basedir: __dirname });
+  addSessionProgressCallback((data) => {
+    GraphAILogger.info(data);
+  });
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)